lib/router/bindDefaultHandlers.js

```diff
@@ -77,7 +81,8 @@ module.exports = function(sails) {
       // Log a message and try to use `res.send` to respond.
       try {
         sails.log.verbose('A request did not match any routes, and no `res.notFound` handler is configured.');
-        res.send(404);
+        res.status(404);
+        res.send('Not found');
```

Comment:
Maybe this? 

https://github.com/Shyp/sails/blob/upgrade-express/lib/router/res.js#L34-L37